### PR TITLE
*: Cut v0.43.0

### PR DESCRIPTION
**Pull request by simonpasquier**
cc @lilic @brancz @s-urbaniak @paulfantom

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
-* [BUGFIX] set pkg/apis/monitoring tag to v0.42.1 (#3531)
+## 0.43.0 / 2020-10-26
+
+This release introduces a new `AlertmanagerConfig` CRD that allows to split the
+Alertmanager configuration in different objects. For now the CRD only supports
+the PagerDuty, OpsGenie and webhook receivers, [other
+integrations](https://github.com/prometheus-operator/prometheus-operator/issues?q=is%3Aissue+is%3Aopen+%22receiver+type%22)
+will follow in future releases of the operator. The current version of the CRD
+is `v1alpha1` meaning that the feature is not yet considered stable and the API
+is subject to change in the future.
```
Comment (lilic) [inline CHANGELOG.md:9]: Maybe we can add: "But feedback on testing is welcome!" wdyt?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+
+* [CHANGE] Use a single reloader sidecar (instead of 2) for Prometheus. The `--config-reloader-image` flag is deprecated and will be removed in a future release (not before v0.45.0). Make sure to start the operator with a version of `--prometheus-config-reloader` that is at least `v0.43.0` otherwise the Prometheus pods will fail to start. #3457
```
Comment (lilic) [inline CHANGELOG.md:11]: Should we make a **warning** here in bold that its going to be deprecated, I always notice a lot of people just glance at this?

**Review by lilic — COMMENTED** [review]
Are you maybe missing a run of `make generate`?

File: CHANGELOG.md (hunk 1 above)
Comment (simonpasquier) [reply CHANGELOG.md]: sure, I've rephrased it.

File: CHANGELOG.md (hunk 2 above)
Comment (simonpasquier) [reply CHANGELOG.md]: done

**Review by lilic — APPROVED** [review]
lgtm 👍 🎉

**Review by paulfantom — APPROVED** [review]
/LGTM :shipit:

**Comment by lilic** [comment]
> simonpasquier changed the base branch from master to release-0.43 16 minutes ago

@simonpasquier we merge to master first, was it maybe a mistake? We then from master once PR is merged create the new branch and tag.

**Comment by simonpasquier** [comment]
@lilic I kind of messed up things because I created the [`pkg/apis/monitoring/v0.43.0`](https://github.com/prometheus-operator/prometheus-operator/tree/pkg/apis/monitoring/v0.43.0) tag before #3621 was merged. And because this PR changed the `pkg/apis/monitoring/go.mod` file, I would have to create a new `pkg/apis/monitoring/v0.43.1` tag which would be confusing.

**Comment by lilic** [comment]
@simonpasquier no worries, just lets make sure to finish up release soon before the branches diverge further, e.g. merge the 0.43 into master. Thanks!

**Pull request merged**